pitch.md

```diff
@@ -1,20 +1,76 @@
-# Project Title (replace me)
+# LED composing (replace me)
 
 ## Describe your project in five words
 
+Easily compose LED color effects
+
 ## Audience
 _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+With my language I am trying to help the casual "maker" or LED user more easily
+design LED compositions and color effects without having to know a lot of CS
+or C coding. Currently, most users of LED strip lights use the default remote 
```

Comment:
I think this is a strong motivating factor for your project. Part of a DSL is to make life easier for the people in your domain to do their tasks.